tendrl/monitoring_integration/manager/__init__.py

```diff
@@ -0,0 +1,142 @@
+import __builtin__
+import traceback
+import json
+
+import signal
+import maps
+import gevent
+import os
+
+from tendrl.monitoring_integration.grafana import utils
+from tendrl.monitoring_integration.grafana import exceptions
+from tendrl.monitoring_integration.grafana import dashboard
+from tendrl.monitoring_integration.grafana import datasource
+from tendrl.commons import manager as common_manager
+from tendrl import monitoring_integration
+from tendrl.monitoring_integration import sync
+from tendrl.commons import TendrlNS
+from tendrl.commons.utils import log_utils as logger
+
+
+class MonitoringIntegrationManager(common_manager.Manager):
+
+    def __init__(self):
+
+        self._complete = gevent.event.Event()
+        super(
+            MonitoringIntegrationManager,
+            self
+        ).__init__(
+            sync.MonitoringIntegrationSdsSyncThread()
+        )
+
+    def start(self):
+        _upload_default_dashboards()
+
+    def stop(self):
+        pass
+
+
+def _upload_default_dashboards():
+
+        monitoring_integration_manager = MonitoringIntegrationManager()
+
+        # Creating Default Dashboards
+        dashboards = []
+        config = utils.get_conf("/etc/tendrl/monitoring-integration/"                    
+                               + "monitoring-integration.conf.yaml")
+        setattr(NS, "conf", config)
```

Comment:
remove L46-48, I commented in last review this is not required

only do this on L124 https://github.com/Tendrl/node-monitoring/blob/master/tendrl/node_monitoring/manager/__init__.py#L31  (NS.monitoring in your case)  and directly use NS.config wherever required